src/modules/app/StateType.js

```diff
@@ -31,9 +31,7 @@ export type CategoryRouteStateType = {|
   +children: $ReadOnly<{ [path: PathType]: $ReadOnlyArray<PathType> }>
 |} | {|
   +status: 'languageNotAvailable',
-  +depth: number,
-  +city: string,
-  +language: string,
+  ...CategoryRouteConfigType,
```

Comment:
Yes, we need it for the `refreshProps` in the containers if the language is not available

Reply:
It can happen, that we don't have a path! For example if you change the language of a subsubscreen and the screen doesn't have the targeted language available. Then there's no path that we can set here.

Reply:
So it was actually on purpose, that we cannot pull-to-refresh on a languageNotAvailable screen

Reply:
Okay what should we do then?

Reply:
Why did you need the change?

Reply:
Flow I think, but will have a look again. To display an error refreshProps are needed. (needed in the containers)

Reply:
Yeah, that's probably the issue. The problem is that we cannot reliably define refreshProps for that special case 🤔
Maybe you can make path optional in the "languageNotAvailable" status, remove the annotated usage in morphcontentlanguage and make refreshprops optional for the error status and only define refreshprops if we have a path in the languageNotAvailable status.

Reply:
All right. This mustn't happen anyway, since a language can not be marked as not available if the languages are not available... So I think it does not really matter in any way

Reply:
I don't really get what you want to say here :see_no_evil:

Reply:
The case that the state of the route is `languageNotAvailable` while the state of languages is `error` or `loading` should not be possible. For a language it can only be decided whether it's available if the languages are fetched and `ready`. So since the previously described state should not occur, it does not really matter what we do in the code to satisfy flow.